### PR TITLE
Hotfix/minikube 1.23

### PR DESCRIPTION
**Pull request by alessandrolomanto**
Updated READMEs, Kubernetes Version v1.23.1 and Fury Modules.

### CODE REVIEW

**Review by ralgozino — CHANGES_REQUESTED** [review]
Hi @alessandrolomanto ! thank you for taking care of this! ❤️ 

there's a misalignment in the module versions that you've used

File: fury-on-minikube/Furyfile.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 versions:
-  monitoring: v1.12.2
-  logging: v1.8.0
-  ingress: v1.10.0
+  monitoring: v1.14.2
+  logging: v2.0.1
+  ingress: v1.12.2
```
[the remark this answers is not in the document]
Comment (ralgozino) [reply fury-on-minikube/Furyfile.yml:4]: Or alternatively, you can get the Furyfile using `furyctl` directly:

```bash
furyctl init --version v1.23.1
```

**Review by alessandrolomanto — COMMENTED** [review]
Minikube tutorial is updated.
Kubernetes v1.23.1
KFD v1.23.1

**Review by ralgozino — APPROVED** [review]
lgtm, thank you @alessandrolomanto !

**Pull request merged**